jwala-common/src/main/java/com/cerner/jwala/common/properties/ApplicationProperties.java

```diff
@@ -99,7 +99,15 @@ public static int size() {
     }
 
     private void init() {
-        String propertiesFile = System.getProperty(PROPERTIES_ROOT_PATH) + "/" + PROPERTIES_FILE_NAME;
+
+        String propertiesRootPath = System.getProperty(PROPERTIES_ROOT_PATH);
+        if (propertiesRootPath == null) {
```

Comment:
I actually prefer forcing unit tests to declare their own Properties Root Path. I thought it was good practice not to provide default properties.